client/extensions/woocommerce/state/ui/payments/methods/test/reducer.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer, { initialState } from '../reducer';
+import {
+	openPaymentMethodForEdit,
+	closeEditingPaymentMethod,
+	cancelEditingPaymentMethod,
+	changePaymentMethodField,
+} from '../actions';
+
+const siteId = 123;
+
+describe( 'reducer', () => {
+	describe( 'openPaymentMethodForEdit', () => {
+		it( 'when the method has a server-side ID', () => {
```

Comment:
You're missing a section of sentence, ideally tests would read `reducer openPaymentMethodForEdit [should do something] when the method has a server-side ID`